p2pserver/net/netserver/netserver.go

```diff
@@ -268,9 +268,9 @@ func (this *NetServer) Connect(addr string, isConsensus bool) error {
 	defer this.Unlock()
 	connCount := this.GetOutConnectingListLen()
 	if connCount > config.DefConfig.P2PNode.MaxConnOutBound {
-		log.Warnf("out connections(%d) reach the max limit(%d)", connCount,
+		log.Warnf("Connect: out connections(%d) reach the max limit(%d)", connCount,
```

Comment:
AFAIK, `GetOutConnectingListLen` returns the number of nodes in handshaking state, not the total number of connected nodes.